test/extended/templates/templateservicebroker_bind.go

```diff
@@ -0,0 +1,119 @@
+package templates
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/pborman/uuid"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/authentication/user"
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/templateservicebroker/openservicebroker/api"
+	"github.com/openshift/origin/pkg/templateservicebroker/openservicebroker/client"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Conformance][templates] templateservicebroker bind test", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		tsbOC               = exutil.NewCLI("openshift-template-service-broker", exutil.KubeConfigPath())
+		portForwardCmdClose func() error
+
+		cli                = exutil.NewCLI("templates", exutil.KubeConfigPath())
+		instanceID         = "aadda50d-d92c-402d-bd29-5ed2095aad2c"
+		bindingID          = uuid.NewRandom().String()
+		serviceID          = "d261a5c9-db37-40b5-ac0f-5709e0e3aac4"
+		fixture            = exutil.FixturePath("testdata", "templates", "templateservicebroker_bind.yaml")
+		clusterrolebinding *authorizationapi.ClusterRoleBinding
+		brokercli          client.Client
+		cliUser            user.Info
+	)
+
+	g.BeforeEach(func() {
+		var err error
+
+		brokercli, portForwardCmdClose = EnsureTSB(tsbOC)
+
+		cliUser = &user.DefaultInfo{Name: cli.Username(), Groups: []string{"system:authenticated"}}
+
+		// enable unauthenticated access to the service broker
+		clusterrolebinding, err = cli.AdminAuthorizationClient().Authorization().ClusterRoleBindings().Create(&authorizationapi.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: cli.Namespace() + "templateservicebroker-client",
+			},
+			RoleRef: kapi.ObjectReference{
+				Name: bootstrappolicy.TemplateServiceBrokerClientRoleName,
+			},
+			Subjects: []kapi.ObjectReference{
+				{
+					Kind: authorizationapi.GroupKind,
+					Name: bootstrappolicy.UnauthenticatedGroup,
+				},
+			},
+		})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		stdout, _, err := cli.Run("process").Args("-f", fixture, "-p", "NAMESPACE="+cli.Namespace()).Outputs()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = cli.AsAdmin().Run("create").Args("-f", "-").InputString(stdout).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
why not use new-app?

Reply:
Previously there was a matching "delete", but I took it out.  Will change.